packages/wrangler/src/index.ts

```diff
@@ -1109,7 +1109,7 @@ export async function main(argv: string[]): Promise<void> {
 		// key to fetch) or flags
 
 		try {
-			const configPath = args.config ?? findWranglerConfig(process.cwd());
+			const configPath = resolveWranglerConfigPath(args);
 			const rawConfig = readRawConfig(args.config);
```

Comment:
This logic doesn't look right (not a change in your PR). I _think_ we should be passing `configPath` to `readRawConfig`? `args.config` won't always be defined